atox/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,120 @@
+<resources>
+    <string name="accept">Akzeptieren</string>
+    <string name="add">Hinzufügen</string>
+    <string name="add_contact">Kontakt hinzufügen</string>
+    <string name="add_contact_message_error_empty">Nachricht darf nicht leer sein</string>
+    <string name="confirm">Bestätigen</string>
+    <string name="cancel">Abbrechen</string>
+    <string name="connecting">Verbindet…</string>
+    <string name="contact_typing">tippt…</string>
+    <string name="copied">Kopiert</string>
+    <string name="copy_atox">Kopieren</string> <!-- androidx.preference:preference has stolen copy -->
+    <string name="create">Erstellen</string>
+    <string name="create_a_tox_profile">Tox Profil erstellen</string>
```

Comment:
We have "Tox-ID", which is good. Then we probably should have "Tox-Profil".
So: `s/Tox Profil/Tox-Profil/g`